pkg/daemon/controller/controller.go

```diff
@@ -153,6 +153,20 @@ func (sc *ServiceController) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(sc)
 }
 
+// Bootstrap starts all required Windows services on this instance. This will result in a node object being created
+func (sc *ServiceController) Bootstrap(services []servicescm.Service) error {
+	bootstrapServices := []servicescm.Service{}
+	for _, svc := range services {
+		if svc.Bootstrap {
+			bootstrapServices = append(bootstrapServices, svc)
+		} else {
+			// services are pre-sorted by priority, with all bootstrap services ordered towards the front of the slice
+			break
+		}
```

Comment:
reduce indentation by handling the negative case first
the extra else is unnecessary 
```
if !svc.Bootstrap {
  break
}
bootstrapServices = append(bootstrapServices, svc)
```

Reply:
Logically this should be done at the same time as
```
cmData, err := servicescm.DecodeAndUnmarshall(encodedCMData)
```